src/webaudio.js

```diff
@@ -286,7 +286,10 @@ WaveSurfer.WebAudio = {
         // close the audioContext if it was created by wavesurfer
         // not passed in as a parameter
         if (!this.params.audioContext) {
-            this.ac.close();
+            // check if browser supports AudioContext.close()
+            if(typeof this.ac.close() !== 'undefined'){
```

Comment:
can you add spaces around the parentheses like line 288? Weird our style checker didn't notice it, I thought it did before.